src/main/java/com/cairnfg/waypoint/authorization/endpoints/account/DeleteAccountByIdEndpoint.java

```diff
@@ -0,0 +1,61 @@
+package com.cairnfg.waypoint.authorization.endpoints.account;
+
+import com.cairnfg.waypoint.authorization.endpoints.ErrorMessage;
+import com.cairnfg.waypoint.authorization.service.AccountService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.transaction.Transactional;
+import java.util.NoSuchElementException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+@Slf4j
+@RestController
+@Tag(name = "Account")
+public class DeleteAccountByIdEndpoint {
+
+  private final AccountService accountService;
+
+  public DeleteAccountByIdEndpoint(AccountService accountService) {
+    this.accountService = accountService;
+  }
+
+  @Transactional
+  @DeleteMapping("/api/account/{accountId}")
```

Comment:
The requestMapping path should be stored in a `public static final String` constant called `PATH` to match the paradigm. This will allow us to test our API endpoints in Integration testing later on without having to copy over strings